cunumeric/array.py

```diff
@@ -509,7 +659,7 @@ def flags(self) -> Any:
         for C-style contiguous arrays or ``self.strides[0] == self.itemsize``
         for Fortran-style contiguous arrays is true.
         """
-        return self.__array__().flags
+        return self._flags
```

Comment:
is it guaranteed that the ndarray wrapped in an `EagerArray` has the same flags as this? I don't see any code that does that.